src/xdocs/config_javadoc.xml

```diff
@@ -1632,6 +1633,21 @@ public boolean isSomething()
 &lt;/module&gt;
         </source>
         <p>
+          Example of {@inheritDoc} without summary.
+        </p>
+        <source>
```

Comment:
please place this example right under example of default configuration.
In this example please place method with empty javadoc and make a trailing comment "//violation " to clearly show user cases how Check is working on default config